goentri.duda.json

```diff
@@ -0,0 +1,34 @@
+{
+    "providerId": "goentri.com",
+    "providerName": "Entri",
+    "serviceId": "duda",
+    "serviceName": "Duda",
+    "version": 1,
+    "logoUrl": "https://cdn.goentri.com/logo.svg",
+    "description": "Allows user to easily set up domain using Entri",
+    "syncPubKeyDomain": "goentri.com",
+    "syncRedirectDomain": "api.goentri.com, goentri.com,entri.com",
+    "variableDescription": "webSiteHost is the customer website host",
+    "sharedProviderName": true,
+    "hostRequired": true,
```

Comment:
So both templates are for subdomains? This one looks more like meant for APEX, but the CNAME confuses me.